app/views/transactions.html

```diff
@@ -103,6 +103,7 @@ <h3 style="margin: 20px;" ng-if="transactionsSelected.length < defaultBulkRerunL
                   <th>Channel</th>
                   <th>Status</th>
                   <th>Request Time</th>
+                  <th>Client</th>
```

Comment:
Having the client on the end looks a little bit out of place. Lets keep the request time at the end and move the client next to the 'Channel' column